recipes/surpyvor/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.5.0" %}
+
+package:
+  name: surpyvor
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/surpyvor/surpyvor-{{ version }}.tar.gz
+  sha256: 58b07eb9848e0b5d38a56f33062bc0e8b06366c58529923cef8aacfc3bd98fd6
+
+build:
+  entry_points:
+    - surpyvor=surpyvor.surpyvor:main
+  noarch: generic
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+
+  run:
+    - python >=3
+    - bcftools
+    - tabix
+    - survivor
+    - numpy
+    - cyvcf2
+    - matplotlib
+    - matplotlib-venn
+    - upsetplot
+
+test:
+  imports:
+    - surpyvor
+
+  commands:
+    - surpyvor --help
+
+about:
+  home: https://github.com/wdecoster/surpyvor
+  license: MIT License
+  summary: 'Evaluating, merging and plotting SV vcf files'
+  license_family: MIT
```

Comment:
If the license file is in the tarball, you can consider it adding it with `license_file`.

Also, you can add yourself as a maintainer if you want and add the `doi` of your publicatiton, e.g.:
```
extra:
  recipe-maintainers:
    - wdecoster
  identifiers:
    - doi:10.1101/gr.244939.118
```

Reply:
Thanks for the suggestion, will do! The license file is not included in the tarball though. I can add it next to the meta.yaml if that would be an added value?

Reply:
Yes, that will work too. Just copy the license file from your repo to the recipe directory and use `license_file: LICENSE`.